### PR TITLE
Update microservice.R for supporting jsonData input in R

### PR DESCRIPTION
**Pull request by LueJian**
Update functions in microservie.R to support "jsonData" input format like Python wrapper.

**What this PR does / why we need it**:
In R, we can use SeldonMessage with not only ndarray format, but also jsonData fromat now!
An example of "jsonData" input format : {"jsonData":{"key1":value1, "key2":value2}}

**Which issue(s) this PR fixes**:

Fixes #

Extending the concept from #595, supporting other language wrappers - R wrapper.

**Special notes for your reviewer**:
[Example] Testing it with a simple transformer-input component, just returning input as its output.
jsonData input format testing

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

**Comment by seldondev** [comment]
Hi @LueJian. Thanks for your PR.

I'm waiting for a [SeldonIO](https://github.com/orgs/SeldonIO/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/SeldonIO/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **ryandawsonuk**
You can assign the PR to them by writing `/assign @ryandawsonuk` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ryandawsonuk — APPROVED** [review]
/lgtm
/approve

**Comment by seldondev** [comment]
Mon Oct 19 15:42:05 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2544/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2544 --build=2

**Comment by seldondev** [comment]
Mon Oct 19 15:42:15 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2544/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2544 --build=1

**Pull request merged**
